docs/stdlib/bytes.rst

```diff
@@ -136,19 +148,48 @@ Bytes
 ---------
 
 .. eql:function:: std::to_bytes(s: str) -> bytes
+                  std::to_bytes(val: int16) -> bytes
+                  std::to_bytes(val: int32) -> bytes
+                  std::to_bytes(val: int64) -> bytes
+                  std::to_bytes(val: int64) -> bytes
+                  std::to_bytes(val: uuid) -> bytes
 
     :index: encode stringencoder
 
     .. versionadded:: 4.0
 
-    Converts a :eql:type:`str` value to :eql:type:`bytes` using
-    UTF-8 encoding.
+    Converts a given value into binary representation as :eql:type:`bytes`.
+
+    The strings get converted using UTF-8 encoding:
 
     .. code-block:: edgeql-repl
 
         db> select to_bytes('テキスト');
         {b'\xe3\x83\x86\xe3\x82\xad\xe3\x82\xb9\xe3\x83\x88'}
 
+    The integer values are encoded as big-endian (most significant bit comes
+    first) byte strings:
+
+    .. code-block:: edgeql-repl
+
+        db> select to_bytes(<int16>31);
+        {b'\x00\x1f'}
+        db> select to_bytes(<int32>31);
+        {b'\x00\x00\x00\x1f'}
+        db> select to_bytes(123456789123456789);
+        {b'\x01\xb6\x9bK\xac\xd0_\x15'}
+
+    The UUID values are converted to the underlying 16-byte string:
```

Comment:
```suggestion
    The UUID values are converted to the underlying string of 16 bytes:
```

(otherwise one might think we are sending the string representation).